Dockerfile.armhf

```diff
@@ -0,0 +1,28 @@
+FROM ghcr.io/linuxserver/baseimage-alpine:arm32v7-3.16
```

Comment:
Should we skip armhf since we dropped it officially?

Reply:
I guess removing it rather than releasing and then deprecating in the near future (if necessary) would be better. I don't know how necessary it would be to deprecate, since fail2ban is just a python package. I don't mind either way.

Reply:
For now we'll keep it. As long as Alpine has armhf available, and python3 is available on armhf.